app/scrapers/twitter.py

```diff
@@ -0,0 +1,24 @@
+import requests
+
+
+class Twitter():
+    """docstring for Twitter"""
+
+    def __init__(self):
+        self.loklakURL = 'http://api.loklak.org/api/search.json?\
+                            timezoneOffset=-330&q='
+
+    def search(self, query, count):
+        """ Makes a GET request to Loklak API and returns the URLs
+        Returns: urls (list)
+                [[Title1,url1], [Title2, url2],..]
+        """
+        url = self.loklakURL+query
+
+        responses = requests.get(url).json()
+
+        tweets = []
+        for response in responses['statuses']:
+            tweets.append({'link': response['link'], 'tweet': response['text']})
```

Comment:
Does downstream expect the dict keys to be ('title', 'link') like [all other scrapers](https://github.com/fossasia/query-server/blob/master/app/scrapers/google.py#L26)?

Reply:
Do we want to print the results like the other scrapers do?

Reply:
Should I change it back to link and title?

Reply:
We need @ParthS007 to weigh in on that question.  Perhaps it is OK as is.